extensions/objects/models/objects.py

```diff
@@ -1620,10 +1634,14 @@ def normalize(cls, raw):
 
         Raises:
             Exception. Given arg is not of type str.
+            Exception. Given arg is not in valid JSON format.
         """
         if not isinstance(raw, str):
             raise Exception('Expected string received %s of type %s' % (
                 raw, type(raw))
             )
 
-        return json.loads(raw)
+        try:
+            return json.loads(raw)
+        except json.decoder.JSONDecodeError as e:
+            raise Exception('Expect %s to be a valid JSON' % (raw)) from e
```

Comment:
Why is this added?

Reply:
I added it to better explain the error raised by the `json.loads()` method when the json to be loaded is not valid. 
I had this error before: `Expecting value: line 1 column 1 (char 0)` in my console and it took me a while to understand that the data passed to the json.loads() method was not valid.

Reply:
Okay, makes sense.

Reply:
should i add it again?